NuRadioReco/utilities/noise.py

```diff
@@ -474,27 +401,28 @@ def __triggering_strided(self):
             #self.max_amp = max(squared_mean.max(), self.max_amp)
             self.max_amp = max(squared_mean.max(), self.max_amp)
             if True in (squared_mean > self.threshold):
-                print(f"triggered at beam {iBeam}")
+                logger.info(f"triggered at beam {iBeam}")
                 if(self.debug):
                     import matplotlib.pyplot as plt
                     fig, ax = plt.subplots(5, 1, sharex=True)
                     for iCh in range(self.n_channels):
                         ax[iCh].plot(self._traces[iCh])
-                        print(f"{self._traces[iCh].std():.2f}")
+                        logger.info(f"{self._traces[iCh].std():.2f}")
                     ax[4].plot(self._phased_traces[iBeam])
                     fig.tight_layout()
                     plt.show()
                 return True
         return False
 
-    def generate_noise_optimised(self, debug=False):
+
+    def generate_noise(self, phasing_mode="slice", trigger_mode="binned_sum", debug=False):
```

Comment:
can you document the two modes here? phasing_mode="slice", trigger_mode="binned_sum"

Reply:
ok done